File: 107598019/HW2/Software Architecture_MySQL/Software Architecture/usecase/editCourse/EditCourseInput.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Software_Architecture.usecase.editCourse
{
    public interface EditCourseInput
    {
        string getOldTitle();

        void setOldTitle(string oldTitle);

        string getTitle();

        void setTitle(string title);

        string getDescription();

        void setDescription(string description);

        string getSuitable();

        void setSuitable(string suitable);

        string getPrice();

        void setPrice(string price);

        string getNotice();

        void setNotice(string notice);

        string getOther();

        void setOther(string other);
    }
}

File: 107598038/HW2/src/main/src/input/editCourse/editCourseInput.java
package input.editCourse;

import input.Input;

public interface editCourseInput extends Input {
    public void setID(int ID);
    public void setPrice(int price);
    public void setNumberOfPeople(int numberOfPeople);
    public void setName(String name);
    public void setNote(String note);
    public void setRemark(String remark);
    public void setSuitable(String suitable);
    public void setDescription(String description);

    public int getID();
    public int getPrice();
    public int getNumberOfPeople();
    public String getName();
    public String getNote();
    public String getRemark();
    public String getSuitable();
    public String getDescription();
}
